src/wp-admin/includes/template.php

```diff
@@ -1714,6 +1751,10 @@ function do_settings_sections( $page ) {
 		echo '<table class="form-table" role="presentation">';
 		do_settings_fields( $page, $section['id'] );
 		echo '</table>';
+
+		if ( '' !== $section['after_section'] ) {
+			echo $section['after_section'];
+		}
```

Comment:
Shouldn't `'after_section'` HTML be escaped before echoing out to the browser?

Reply:
It would make sense to allow developers pass any HTML content in `after_section`. Are you suggesting we should only allow certain HTML tags here?

I also thought about [escaping the `title`](https://github.com/WordPress/wordpress-develop/blob/6.0/src/wp-admin/includes/template.php#L1696), but that could break things for anyone using HTML tags (e.g. `<SPAN>`) in the title.

Reply:
Escaping the title is out of scope for this ticket.

I'm thinking about security of allowing any raw HTML to be echoed out to the browser. 

I'd suggest considering passing the `'before_section'` and `'after_section'` HTML through `wp_kses_post()`.

Reply:
Both `'before_section'` and `'after_section'` are now escaped using `wp_kses_post()` before rendering.